CMakeLists.txt

```diff
@@ -114,16 +138,28 @@ install(
 
 set(API_JSON_FILE ${PROJECT_BINARY_DIR}/unified_runtime.json)
 
-# generate source from the specification
+# Generate source from the specification
 add_custom_target(generate
     WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/scripts
     COMMAND ${Python3_EXECUTABLE} run.py --api-json ${API_JSON_FILE}
     COMMAND ${Python3_EXECUTABLE} json2src.py --api-json ${API_JSON_FILE} ${PROJECT_SOURCE_DIR}
 )
 
-# generate source and check for uncommitted diffs
+# Generate source and check for uncommitted diffs
 add_custom_target(check-generated
     WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
     COMMAND git diff --exit-code
     DEPENDS generate
 )
+
+# Add code formatter target
+add_custom_target(cppformat)
```

Comment:
I think we should fail the build if format generates something.

Reply:
We should probably have some DEVELOPER mode for that

Reply:
I was thinking about adding `DEPENDS cppformat` to the `check-generated` target.

Reply:
Done. Or should I also add a developer mode?

Reply:
No, I think it's ok. But let's not enable this until we actually have the docker script integrated.

Reply:
Done. I've commented it.